generator/wrapper_gen.py

```diff
@@ -245,6 +251,240 @@ def daal_generate_shuffled_indices(idx, random_state):
     c_generate_shuffled_indices(data_or_file(<PyObject*>idx), data_or_file(<PyObject*>random_state))
 
 
+def get_gbt_model_from_lgbm(model: Any) -> Any:
+    class Node:
+        def __init__(self, tree: Dict[str, Any], parent_id: int, position: int):
+            self.tree = tree
+            self.parent_id = parent_id
+            self.position = position
+
+    lgb_model = model.dump_model()
+
+    n_features = lgb_model["max_feature_idx"] + 1
+    n_iterations = len(lgb_model["tree_info"]) / lgb_model["num_tree_per_iteration"]
+    n_classes = lgb_model["num_tree_per_iteration"]
+
+    is_regression = False
+    objective_fun = lgb_model["objective"]
+    print("OBJ:   " + objective_fun)
+    if n_classes > 2:
+        if "multiclass" in objective_fun:
+            print("Found multiclass classification")
```

Comment:
not sure that pint should be always available, perhaps we should consider verbose mode enabling.

Reply:
I forgot to remove it, just like another one in 'example' file.
It'll disappear in the next commit

Reply:
all prints are skipped now